pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -119,7 +119,7 @@ export class SystemInfomationCard extends React.Component {
                                     <div id="system_information_hardware_text">{this.state.hardwareText}</div>
                                 </td>
                             </tr>}
-                            {this.state.assetTagText && <tr>
+                            {this.state.assetTagText !== "" && <tr>
```

Comment:
That's correct, hmmm it's really curious how it without this line throws that warning.

Reply:
Note that these do two different things. `assetTagText` is most likely `undefined` for jelle. This is falsy. However, with this PR's change, it would actually evaluate to true, as `undefined !== ""`. Thus this hides the actual issue by actually rendering the `<tr>` -- you should see an "Asset tag" row with an empty value?

So I suppose this syntactical construction with `{ undefined && <tr> ... }` somehow results in a whitespace. Maybe we just need to pull this out into some `assetTagRow` to avoid it?

Reply:
The issue was indeed, that we switched from undefined => "". Still confused that it doesn't render an empty <tr>, but I guess setState might just warn about it and the condition is ok (so it's more of a lint warning).

I've re-adjusted the PR to always set undefined when the string is "", that should fix the issue and also save us from doing a useless setState.